azure-pipelines-docs-integration.yml

```diff
@@ -144,8 +144,8 @@ stages:
             AWS_CONFIG_FILE: $(aws_authkey.secureFilePath)
             # GCP credentials
             GOOGLE_APPLICATION_CREDENTIALS: $(gcp_authkey.secureFilePath)
-            GCP_PROJECT: $(GCP_PROJECT)
-            GCP_BIGQUERY_DATASET: $(GCP_BIGQUERY_DATASET)
+            GE_TEST_GCP_PROJECT: $(GE_TEST_GCP_PROJECT)
```

Comment:
Just confirming that GE_TEST_GCP_PROJECT replaces both GE_TEST_BIGQUERY_PROJECT and GCP_PROJECT?

Reply:
Yes @kenwade4, it was an attempt to consolidate :)